client_test/virtualdefund_test.go

```diff
@@ -61,14 +61,16 @@ func runVirtualDefundIntegrationTest(t *testing.T, messageDelay time.Duration, o
 	clientI, storeI := setupClient(irene.PrivateKey, chainServiceI, broker, logDestination, messageDelay)
 
 	numOfVirtualChannels := uint(5)
-	paidToBob := uint(1)
-	totalPaidToBob := paidToBob * numOfVirtualChannels
+
+	// We can only trigger payments through the payment API now
+	// so we expect to close our virtual channels with no payments to bob
+	totalPaidToBob := uint(0) //paidToBob * numOfVirtualChannels
```

Comment:
We could just update this test with calls to `clientA.Pay` so we're using a non zero amount. Wasn't sure if calls to `Pay` belong in this test or should just live in the new `payment_test` integration test.